Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -312,7 +312,7 @@ Wiimote::Wiimote(const unsigned int index)
 
 std::string Wiimote::GetName() const
 {
-  return std::string("Wiimote") + char('1' + m_index);
+  return std::string("Wii Remote") + char('1' + m_index);
```

Comment:
This one may be a config string. (I'm not sure.) This needs to be checked before merging.